<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindowBellmanFord</class>
 <widget class="QDialog" name="GraphWindowBellmanFord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Shortest way between two vertexes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Source ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Destination ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>Bellman-Ford</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>Clean</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_5"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
